rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -261,6 +264,12 @@ public void registerBtcTransaction(Transaction rskTx, BtcTransaction btcTx, int
             return;
         }
 
+        VarInt inputsCounter = new VarInt(btcTxSerialized, 4);
```

Comment:
Add a `btcTxSerialized` length range check before decoding inputs, length of VarInt can be as long as 9 bytes.

Reply:
@juli currently there is no such check in the `co.rsk.bitcoinj.core.VarInt#VarInt(byte[], int)` constructor nor in the `co.rsk.bitcoinj.core.BtcTransaction#parse`. If the length is lower than 9 bytes then it'll throw `java.lang.IndexOutOfBoundsException`. I agree it would be good to check it but I don't want to add any check which didn't existed before for not hardforking the network